lib/registry/registry-client.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license
+ * Copyright 2017 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import * as Dockermodem from 'docker-modem';
```

Comment:
NIT: Would `import type` (rather than just `import`) work here? The only advantage is that `import type` makes it clear to the reader that this is only for TypeScript type checking and will disappear in compiled JavaScript.